docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -1,31 +1,28 @@
-import { getPropTypesFromFile } from '@mui/monorepo/packages/typescript-to-proptypes';
-import {
-  UnionType,
-  InterfaceType,
-} from '@mui/monorepo/packages/typescript-to-proptypes/src/models';
+import { getPropTypesFromFile } from 'typescript-to-proptypes';
```

Comment:
Nitpick: this package seems a bit strangely put together in the MUI Core repo. 🙈 
The `docs` package has a `typescript-to-proptypes` dependency, which does not seem to be used anywhere and we internally define a package with the same name. 🤷 
It just feels "clumsy" when you start looking at it. 🤔

Reply:
The core repo has the same deps on the doc
I think it's a very old legacy dep that is indeed never used